crates/oxc_linter/src/autofix/mod.rs

```diff
@@ -5,26 +5,13 @@ mod fixer;
 pub use fixer::Fixer;
 
 #[derive(Debug)]
-pub struct Fix {
-    pub content: String,
+pub struct Fix<'a> {
+    pub content: &'a str,
     pub span: Span,
 }
 
-impl<'a> Fix {
+impl<'a> Fix<'a> {
```

Comment:
We should add a constructor `fn new<S: AsRef<str>)(...)`.

Reply:
I'm wondering why we should use `AsRef<str>` here?

Reply:
Good point, the API isn't too clear for us right now, let's stick with &str for now.

Reply:
Since we are storing `Cow`, I use `T: Into<Cow<'a, str>>` instead of `AsRef`.